src/pip/_internal/commands/show.py

```diff
@@ -33,6 +36,11 @@ def __init__(self, *args, **kw):
             action='store_true',
             default=False,
             help='Show the full list of installed files for each package.')
+        self.cmd_opts.add_option(
+            '--json',
```

Comment:
I'm thinking me might want to stick to `--format json` like for the `pip list` command.
cc @pypa/pip-committers

Reply:
Sounds good to me!

Reply:
@xavfernandez, @pradyunsg, `--format=json` follows `pip list` options.  For the current format option, `--format=header` mirrors the function name. 

Does that sound about right for the options?

Reply:
Yup with header being the default value :+1: